vmServer/send/pipeline.py

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/python
+"""Script to create the protofile
+
+    It compiles the proto definition and
+    then creates the proto file from the text specified
+    in query1.txt or query2.txt
+"""
+import os
+
+
+os.chdir('proto')
+os.system("python .\\compile_proto.py")
```

Comment:
compile_proto seems to be one line of code that perhaps we can just place here?

Reply:
Change has been made